app/resources/en/translation/signout.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Probate - You&rsquo;ve signed out",
```

Comment:
Following the work in Adam's story PRO-2839 we're currently adding a suffix of '- Apply for Probate' to the end of each title so we'd be better to remove the 'Probate - ' here so we get a title of:

'You've signed out - Apply for Probate' 

rather than:

'Probate - You've signed out - Apply for Probate'